Remove 404'ing links

TanayParikh opened the pull request:
<!-- 👀 Thanks for opening a PR! Read comments like this one to get your PR merged faster. -->
#### Summary
<!-- ✍️ In a sentence or two, describe your changes -->

These references no longer resolve: https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/addHitRegion


This PR…
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

<!-- 👷‍♀️ After submitting, review the results of the "Checks" tab! -->

TanayParikh committed:
Remove 404'ing links

These references no longer resolve: https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/addHitRegion

teoli2003 commented:
If we have deleted the `Canvas.addHitRegion` page, we should delete this one too.

TanayParikh committed:
Update index.md

TanayParikh committed:
Delete index.md

TanayParikh commented:
> If we have deleted the `Canvas.addHitRegion` page, we should delete this one too.

Updated.

TanayParikh committed:
Merge branch 'main' into patch-1

teoli2003 commented:
Thank you! 

In order to _delete_ a page, you have to follow this procedure: https://github.com/mdn/content#deleting-a-document
We store meta documentation with editor names and we have a file storing redirects. This procedure will take care of it.

(Note we want a redirect from /Web/API/MouseEvent/region to /Web/API/MouseEvent to prevent 404.

teoli2003 reviewed (changes requested):
Please use the correct procedure to delete a file.

TanayParikh committed:
Add back file

TanayParikh committed:
Delete file using yarn

yarn content delete /Web/API/MouseEvent/region en-US --redirect web/api/mouseevent

TanayParikh committed:
Revert "Delete file using yarn"

This reverts commit c2b7d0706b8a2a563654565b2d9b1065a7da7abd.

teoli2003 commented on files/en-us/_redirects.txt:
```diff
@@ -6,6 +6,7 @@
 # FROM-URL	TO-URL
 /en-US/docs/-moz-locale-dir(ltr)	/en-US/docs/Web/CSS/:-moz-locale-dir(ltr)
 /en-US/docs/-moz-locale-dir(rtl)	/en-US/docs/Web/CSS/:-moz-locale-dir(rtl)
+/en-US/docs//Web/API/MouseEvent/region	/en-US/docs/web/api/mouseevent
```
Capitalization is wrong: /en-US/docs/Web/API/MouseEvent 

(same below)

TanayParikh replied:
Yep just noticed that as well, not sure what happened. Sorry about that, updated appropriately now.

TanayParikh committed:
Delete using yarn

yarn content delete Web/API/MouseEvent/region en-US --redirect Web/API/MouseEvent

teoli2003 reviewed (approved):
Thank you!

teoli2003 commented:
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:
